src/AddressRegistry/StreetName/StreetNameAddress.cs

```diff
@@ -272,7 +272,8 @@ public void RetireBecauseStreetNameWasRetired()
             }
         }
 
-        private AddressGeometry GetFinalGeometry(
+        public static AddressGeometry GetFinalGeometry(
+            MunicipalityId municipalityId,
```

Comment:
Since the parameter is always the same, I don't see the use of introducing this.

Reply:
The method became static so it can't use the local variable _streetName.MunicipalityId anymore. The reason the method became static is that we want to call it when proposing an address. But an address object is only created when the event is being applied, so we don't have an address object yet. Hence the method was made static. Maybe we should refactor this to a class?